consoleme/routes.py

```diff
@@ -139,13 +143,18 @@ def make_app(jwt_validator=None):
         (r"/api/v2/generate_policy", GeneratePolicyHandler),
         (r"/api/v2/managed_policies/(\d{12})", ManagedPoliciesForAccountHandler),
         (r"/api/v2/managed_policies/(.*)", ManagedPoliciesHandler),
+        (
+            r"/api/v2/templated_resource/([a-zA-Z0-9_-]+)/(.*)",
+            TemplatedResourceDetailHandler,
+        ),
         (
             r"/api/v2/managed_policies_on_role/(\d{12})/(.*)",
             ManagedPoliciesOnRoleHandler,
         ),
         (r"/api/v2/login", LoginHandler),
         (r"/api/v2/login_configuration", LoginConfigurationHandler),
         (r"/api/v2/logout", LogOutHandler),
+        (r"/api/v2/self_service_resource_typeahead", SelfServiceStep1ResourceTypeahead),
```

Comment:
I'm starting to wish we used an `/api/v2/typeahead/[blah]` pattern for these endpoints 😞